nominal/__init__.py

```diff
@@ -1,13 +1,48 @@
-import os
+from .nominal import (
+    set_default_connection,
+    get_default_connection,
+    upload_dataset_from_pandas,
+    upload_dataset_from_polars,
+    upload_dataset,
+    get_dataset_by_rid,
+    update_dataset,
+    create_run,
+    get_run_by_rid,
+    search_runs,
+    update_run,
+    add_dataset_to_run,
+    list_datasets_for_run,
+    add_attachment_to_run,
+    list_attachments_for_run,
+    upload_attachment,
+    get_attachment_by_rid,
+    update_attachment,
+    save_attachment,
+)
+from .sdk import Dataset, Run, Attachment, NominalClient
 
-# Allows:
-# import nominal as nm
-# nm.cloud ...
-# nm.data ...
-from . import cloud as cloud
-from . import data as data
-from .nominal import Dataset as Dataset
-from .nominal import Ingest as Ingest
-from .nominal import Run as Run
-
-os.environ["NOMINAL_BASE_URL"] = "https://api-staging.gov.nominal.io/api"
+__all__ = [
+    "set_default_connection",
+    "get_default_connection",
+    "upload_dataset_from_pandas",
+    "upload_dataset_from_polars",
+    "upload_dataset",
```

Comment:
naming suggestions:
* `upload_polars`
* `upload_pandas`
* `upload_csv`
I broadly like the idea of the `upload_` methods being super specific, so if we e.g support new file types, they would all be new individual methods (`upload_parquet`, etc).